README.md

```diff
@@ -0,0 +1,62 @@
+# Front End Capstone - Open Table: Proxy Server 
+
+Project resembles a clone of Open Table:
+
+4 Primary Components Rendering:
+
+  - About/Photos Component 
+
+https://github.com/thesilverspoon/about-photos-component
+  
+  - Reservations Component
+
+https://github.com/thesilverspoon/reservations-component
+
+  - Menus Component
+
+https://github.com/thesilverspoon/menu-component
+
+  - Reviews Component
+
+https://github.com/thesilverspoon/reviews-component
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine.
+
+### Prerequisites
+
+What things you need to install the software and how to install them
+
+```
+MongoDB
+PostgreSQL
+NodeJS
+
+```
+
+### Initial Installations
+
+
+```
+For each of the 4 Components:
+
+Clone each component then add into 4 separate folders in this proxy repo.
+
+Run 'npm install' all dependencies for each.
+
+Run 'npm run build' for production bundle.js file for each.
```

Comment:
hi Michael, It's nice to have this, but I think the command might be different for the different components, might want to check the `package.json`s for each component